session1/examples/types-interface.ts

```diff
@@ -0,0 +1,31 @@
+export interface Address {
+    apartment: number;
+    street: string;
+    city: string;
+    state: string;
+    pin: number;
+}
+
+let firstName: string = "John";
+let lastName: string = "Doe";
+let age: number = 22;
+let address: Address = {
+    apartment: 101,
+    street: "Fort Street",
+    city: "Mumbai",
+    pin: 400001,
+    state: "Maharashtra",
+};
+
+const getAddress = (address:  Address): Address => {
```

Comment:
Use template literals in ES6 and not ES5 syntax.

Reply:
TBD